src/features/search/styled.js

```diff
@@ -8,14 +8,13 @@ export const Label = styled.label`
   border: 1px solid ${({ theme }) => theme.color.grey};
   display: flex;
   align-items: center;
-  margin: 23px 200px;
+  margin-left: auto
```

Comment:
tutaj nawet nie jest potrzebny margines z tego co patrzyłem ponieważ komponent wyżej ma justify-content:space-between;

Reply:
Widzę, w następnych plikach, że tam jest pozmieniane w innych plikach więc nie ma tematu ;)

Reply:
ogólnie to jest taki trik, który znalazłem na necie, bo w flexie nie ma czegoś takiego jak justify-item, więc nie można sobie przerzucić jednego elementu flexa na prawą stronę, a ten search powinien być przyklejony do prawej krawędzi wg mnie i ten margin-left: auto tak właśnie działa